Src/BootCamp.Chapter/Armor.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    public class Armor : Item
+    {
+        float _defence;
+        public float GetDefence()
+        {
+            return _defence;
+        }
+
+        public Armor(string name, decimal price, float weight, float defence):base(name, price, weight)
+        {
+            _defence = defence;
+        }
+
+        public static Armor NullArmor()
```

Comment:
This feels really bad. Null should be null and nothing else. Don't do this. If you think you need this you probably have design issues.

Reply:
I removed NullArmor and NullItem